app/1.0/docs/devguide/properties.md

```diff
@@ -817,96 +343,16 @@ generated setter of the convention <code>\_set<var>Property</var>(value)</code>.
 ```
 
 For more on read-only properties and data binding, see
-[Property change notification and two-way binding](data-binding#property-notification).
-
-## Computed properties {#computed-properties}
-
-Polymer supports virtual properties whose values are calculated from other
-properties.
+[How data flow is controlled](data-system#data-flow-control).
```

Comment:
these "How data flow is controlled" links all link to this page. Is that correct? They don't go anywhere.